astoria/common/config.py

```diff
@@ -48,11 +48,27 @@ class Config:
         extra = "forbid"
 
 
+class WiFiInfo(BaseModel):
+    """"WiFI credentials"""
+
+    enabled: bool = True
+    ssid: Optional[str]
+    psk: Optional[str]
+    region: Optional[str]
+    interface: str = "wlan0"
```

Comment:
```suggestion
    """"System settings for WiFi."""

    interface: str = "wlan0"
```

Most of these should be in `Metadata`. See below comments

Reply:
Also, there should definitely not be a default value for interface. That setting is very system specific.